### DISCUSSION TITLE
keyboard mapping

### DISCUSSION THREAD

**Discussion by gabszap** [barry-ran/QtScrcpy#488] [Q&A]
How do I map my keyboard keys to PUBG? (PUBGM 1.5: Ignition)

**Comment by nxwi** [comment]
Use gameforpeace.json
 
![image](https://user-images.githubusercontent.com/51323070/141635420-bdad051c-e9f5-4273-b262-375b38749ad8.png)

**Comment by w4po** [comment]
Hi, Sorry for the late response but I've just made a visual key mapper for QtScrcpy: https://github.com/w4po/ScrcpyKeyMapper
